app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,4 @@
+class ReviewsController < ApplicationController
+
+
+end
```

Comment:
Since our user stories only ask that we go to a book show we won't need a ReviewsController.